File: src/main/scala/com/deswaef/wowscrappie/tools/controller/TierDistributionController.scala
package com.deswaef.wowscrappie.tools.controller

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RequestMethod._

@Controller
@RequestMapping(Array("/tools/tierdistribution"))
class TierDistributionController {

  @RequestMapping(method = Array(GET), value = Array("/t18"))
  def tier18: String = "tools/tier_distribution/t18"

}

File: wowscrappie-web/src/main/java/com/deswaef/wowscrappie/tools/controller/TierDistributionController.java
package com.deswaef.wowscrappie.tools.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;

@Controller
@RequestMapping("/tools/tierdistribution")
public class TierDistributionController {

    @RequestMapping(method = RequestMethod.GET, value = "/t18")
    public String tier18() {
        return "tools/tier_distribution/t18";
    }

}
